src/System.Diagnostics.DiagnosticSource/src/DiagnosticSourceUsersGuide.md

```diff
@@ -125,7 +125,7 @@ Thus the event names only need to be unique within a component.
    most important scenarios involve turning on whole listeners and not needing to filter
    for particular events.    You may need to split
    a source into multiple smaller ones to achieve this, and this is OK.   For example there
-   are both incomming Http request and outgoing Http requests and you may only need one 
+   are both incoming Http request and outgoing Http requests and you may only need one 
```

Comment:
`are both incoming HTTP requests and outgoing HTTP requests and you may only need one`?